### DISCUSSION TITLE
Break out of prose width on an element by element basis in Gatsby/React

### DISCUSSION THREAD

**Discussion by hisnameisjimmy** [tailwindlabs/tailwindcss#6079] [Help] [answered]
I ultimately want to be able to selectively apply prose to elements, however, this is primarily due to the default width. I want certain elements/components to be wider than the default width (for example, tables and images that would benefit from extra width) I've read through [this entire discussion](https://github.com/tailwindlabs/tailwindcss-typography/issues/32), and looked at [this PR](https://github.com/tailwindlabs/tailwindcss-typography/pull/203), but it seems that the solution presented doesn't work for me because the issue is on the [prose class itself](https://github.com/tailwindlabs/tailwindcss-typography/issues/32#issuecomment-927123551), not its children. 

One of the ideas that @RobinMalfait recommends here seems perfect: https://github.com/tailwindlabs/tailwindcss-typography/issues/32#issuecomment-756687596, however, because every item contains just a single element, it also becomes a first or last child, and that means the margins get killed. You can see what I mean on the attached screenshot below.

I feel like I'm so close to being able to do this. Can anyone point to where I'm either missing something or another avenue I should explore?

![CleanShot 2021-11-13 at 13 05 38@2x](https://user-images.githubusercontent.com/740373/141659088-dbf58935-f500-47f9-8e76-1905a61210e7.png)

**Comment by adamwathan** [comment, marked as answer]
Personally I would make sure the tables/images are wrapped in a `div` and throw some negative horizontal margin on that `div` with utility classes.

https://play.tailwindcss.com/2SDxtQSZwW

**Comment by hisnameisjimmy** [comment]
Damn, this is so much more elegant than the absolutely insane workaround that I spent hours on. Another humbling day on the internet, thank you @adamwathan!
